blog/2015/12/call-for-candidates-elections-2016.html

```diff
@@ -0,0 +1,60 @@
+---
+layout: blog
+root: ../../..
+author: Jonah Duckles
+title: "Call for Candidates for the 2016 Steering Committee"
+date: 2015-12-08
+time: "17:00:00"
+category: ["Community", "Elections", "Steering Committee"]
+---
+
+<!-- start excerpt -->
+<p>From February 15-19, Software Carpentry will hold its annual election for the
+Steering Committee of The Software Carpentry Foundation. The inaugural Steering
+Committee has been an incredible team of dedicated community members working to
+position Software Carpentry so that it has a strong foundation and continued
+impact in scientific communities around the world. Please consider giving
+back to the community by standing for election!</p>
+<!-- end excerpt -->
+
+<p>If you are a qualified instructor who has taught at least twice in the past
+two years, or have done non-teaching work for the community, you can both stand
```

Comment:
Can we add a note asking people that have done non-teaching work to contact us if they want to vote? Keep tracking of non-teaching work isn't very easy.